cpplint.py

```diff
@@ -981,6 +976,16 @@
 # This is set by --headers flag.
 _hpp_headers: set[str] = set()
 
+# These headers are excluded from [build/include_subdir], [build/include_order], and
+# [build/include_alpha]
+# The default checks are following
+# - Anything not following google file name conventions (containing an
+#   uppercase character, such as Python.h or nsStringAPI.h, for example).
+# - Lua headers.
+# Default pattern for third-party headers (uppercase .h or Lua headers).
+_third_party_headers_default = r"^(?:[^/]*[A-Z][^/]*\.h|lua\.h|lauxlib\.h|lualib\.h)$"
+_third_party_headers_pattern = re.compile(_third_party_headers_default)
```

Comment:
The default one's a constant so it should still be capitalized.
```suggestion
_THIRD_PARTY_HEADERS_DEFAULT = r"^(?:[^/]*[A-Z][^/]*\.h|lua\.h|lauxlib\.h|lualib\.h)$"
_third_party_headers_pattern = re.compile(_THIRD_PARTY_HEADERS_DEFAULT)
```